go/linux/web.config

```diff
@@ -5,29 +5,31 @@
             <rules>
                 <clear />
 
-                <rule name="/go/linux/12.0/download-keyboards" stopProcessing="true">
-                    <match url="^download-keyboards" />
+                <!-- Redirects for Keyman for Linux 11.0 onward -->
+
+                <rule name="/go/linux/X.0/download-keyboards" stopProcessing="true">
+                    <match url="^(11|12|13)\.0/download-keyboards" />
```

Comment:
Let's make the match on `([1-9][0-9])\.[0-9]` so we don't need to update this every release.